Buildings/Resources/Scripts/Dymola/Experimental/OpenBuildingControl/CDL/Discrete/Examples/DayType.mos

```diff
@@ -1,4 +1,4 @@
-simulateModel("Buildings.Experimental.OpenBuildingControl.CDL.Discrete.Examples.DayType", startTime=-1.8144e+06, stopTime=1.8144e+06, method="dassl", resultFile="DayType");
+simulateModel("Buildings.Experimental.OpenBuildingControl.CDL.Discrete.Examples.DayType", startTime=-1.8144e+06, stopTime=1.8144e+06, method="dassl", resultFile="DayType1");
```

Comment:
The `resultFile` name needs to be the same as the name that is used in the plot command. If there is a name conflict, you can use for example `Examples.DayType` (or any other string). Otherwise, the plot won't display the result.